src/es/components/organisms/checkout/Checkout.js

```diff
@@ -0,0 +1,56 @@
+// @ts-check
+import { Shadow } from '../../web-components-toolbox/src/es/components/prototypes/Shadow.js'
+
+/**
+* @export
+* @class Checkout
+* @type {CustomElementConstructor}
+*/
+export default class Checkout extends Shadow() {
+  constructor (options = {}, ...args) {
+    super({ importMetaUrl: import.meta.url, ...options }, ...args)
+
+    this.checkoutConfigurationListener = (event) => {
+      console.log(event)
+    }
+  }
+
+  connectedCallback () {
+    if (this.shouldRenderCSS()) this.renderCSS()
+    if (this.shouldRenderHTML()) this.renderHTML()
+    this.addEventListener('checkout-configuration', this.checkoutConfigurationListener)
```

Comment:
you likely need to listen to "document.body" instead of "this" if the controller is higher up in the DOM